package.json

```diff
@@ -41,12 +41,15 @@
   },
   "license": "MIT",
   "devDependencies": {
-    "chai": "^2.3.0",
     "cheerio": "^0.19.0",
     "express": "^4.12.4",
     "jade": "^1.10.0",
+    "magicpen-media": "1.5.0",
```

Comment:
Is it being used? Honestly i still don't get it well, but according to their docs it seems the plugin need to be initialized and i don't see it happening in code (?)